fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBDatabase.java

```diff
@@ -766,14 +766,23 @@ private Transaction createTransaction(@Nonnull FDBRecordContextConfig config,
         return transaction;
     }
 
+    /**
+     * Create an {@link FDBDatabaseRunner} for use against this database.
+     * @param contextConfigBuilder options for contexts opened by the new runner
+     * @return a new runner
+     */
+    @Nonnull
+    public FDBDatabaseRunner newRunner(@Nonnull FDBRecordContextConfig.Builder contextConfigBuilder) {
+        return new FDBDatabaseRunnerImpl(this, contextConfigBuilder);
```

Comment:
It's a little weird that one might forget to set the default transaction timeout millis here if one isn't careful...I suppose a "default context config builder" method on a database (or, I suppose, a static "default builder" method on a `FDBRecordContextConfig` that took an `FDBDatabase`) as suggested in a different comment might help with that.

Reply:
This is no longer the case.